test/modules/linear_boltzmann_solvers/transport_keigen/keigenvalue_transport_3d_openmcxs.lua

```diff
@@ -0,0 +1,94 @@
+-- 3D 172G KEigenvalue::Solver test using power iteration and OpenMC MGXS library
+-- Test: Final k-eigenvalue: 1.5030447
+num_procs = 4 
+
+--########## Mesh ##########
+
+-- Cells
+Nx = 5
+Ny = 5 
+Nz = 5 
+
+-- Dimensions
+Lx = 2.0
+Ly = 2.0
+Lz = 2.0
+
+xmesh = {}
+xmin = 0.0
+dx = Lx/Nx
+for i = 1, (Nx+1) do
+  k = i-1
+  xmesh[i] = xmin + k*dx
+end
+
+ymesh = {}
+ymin = 0.0
+dy = Ly/Ny
+for i = 1, (Ny+1) do
+  k = i-1
+  ymesh[i] = ymin + k*dy
+end
+
+zmesh = {}
+zmin = 0.0
+dz = Lz/Nz
+for i = 1, (Nz+1) do
+  k = i-1
+  zmesh[i] = zmin + k*dz
+end
+
+meshgen1 = mesh.OrthogonalMeshGenerator.Create({ node_sets = {xmesh, ymesh, zmesh} })
+mesh.MeshGenerator.Execute(meshgen1)
+
+--########## Materials ##########
+
+materials = {}
+materials[1] = mat.AddMaterial("Fissile Material")
+mat.AddProperty(materials[1], TRANSPORT_XSECTIONS)
+mat.SetProperty(materials[1], TRANSPORT_XSECTIONS, OPENMC_XSLIB, "uo2.h5", 294)
```

Comment:
should 294 be an integer always or could it be 293.25 ?

Reply:
It's read as a float, but, when it's converted to a string, anything after the decimal is dropped. So, 293.25 would not work since the xs data is evaluated at 294. I'll change this to 294.0 in the input.

Reply:
Fixed.